### DISCUSSION TITLE
Wiki.js Features

### DISCUSSION THREAD

**Discussion by Francois19901** [requarks/wiki#6458] [Help / Questions] [answered]
Hello, 

i'm looking to implement one Wiki solution into my compagny. 
There are several answer for which I have not found response on internet. 

Is it possible to export pages/sites/space ? (PDF, XML, etc...)

Is it possible to manage consultation rights on certain pages/sites/space ?

Is it possible to implement Excel files on wiki pages ? If yes, can we modify the files directly on the page ?

Thank you

**Comment by P4mes4n** [comment, marked as answer]
Hi,

1. Its possible to export as PDF using ctrl+p "print as pdf" feature. This already filters out menu items and just prints the actual content. 
2. Im not sure what exactly you mean by "consultation" but if you are talking about read/write permissions for pages, then yes, there is a granular permission system which allows to manage permissions for groups/users on specific pages, paths, assets and so on.
3. Currently not that im aware of. You can upload excel files and put download links in the pages but you cant display or modify it. In V3 it is planned to implement a Excel-Like Editor for this use-case.
